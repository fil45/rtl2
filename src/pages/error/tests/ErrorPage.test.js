import React from "react";
import { render } from "@testing-library/react";
import ErrorPage from "../ErrorPage";

describe("ErrorPage --->", () => {
  it("should render the page", () => {
    const { getByText } = render(<ErrorPage />);
    const errorElement = getByText(/404 ERROR/i);
    expect(errorElement).toBeInTheDocument();
  });

  it("should match snapshot", () => {
    const { asFragment } = render(<ErrorPage />);
    expect(asFragment(<ErrorPage />)).toMatchSnapshot();
  });
});
