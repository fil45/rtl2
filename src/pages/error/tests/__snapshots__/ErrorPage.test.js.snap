// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage ---> should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2>
      404 ERROR
    </h2>
  </div>
</DocumentFragment>
`;
